1. Acceptances.wls

```diff
@@ -2012,115 +2050,115 @@
 (*If[(DirectoryQ["Acceptances"]//ToString)=="False",CreateDirectory["Acceptances"]];*)
 (*If[(DirectoryQ[FileNameJoin[{"Acceptances",GivenExperimentName}]]//ToString)=="False",CreateDirectory[FileNameJoin[{"Acceptances",GivenExperimentName}]]];*)
 (*(*ChoiceDialog["Choose the LLP for which acceptances will be computed:",Join[LLPlistReduced,{"All"}],Appearance\[Rule]"Vertical"\[Rule]{Automatic,3}]*)*)
-(*ivaltest=If[GivenExperiment=="SHiNESS",10^4,1000];*)
+(*ivaltest=If[GivenExperiment=="SHiNESS",10^4,If[GivenExperiment=="SHIFT-muon-chamber",100,1000]];*)
```

Comment:
I'm not sure I understand the syntax correctly, but shouldn't it be like this:

```
(*ivaltest=If[GivenExperiment=="SHiNESS",10^4,If[GivenExperiment=="SHIFT-muon-chamber",100],1000];*)
```

Such that it's 10000 for SHiNESS, 100 for SHIFT, and 1000 otherwise? Also, you mention in the documentation we reduced it to 100 for all experiments, but here it's still 1000 - we should make sure in the end documentation is in sync with the actual code.

Reply:
The syntax is correct because in Mathematica the ifs are: If[condition, trueValue, falseValue], so first checks if the experiment is SHINESS and then if it is not it assigns the "falseValue" which again checks if it is SHIFT. So if the experiments is neither SHINESS or SHIFT it would assign 1000. 

An yes, in the actual code it is 100, I will change it here.

Reply:
Oh I see, makes sense!